### PR TITLE
Added flow observation and aggregate queries.

### PR DESCRIPTION
**Pull request by IdreesInc**
Added flow observation and aggregate queries to the analytics query builder page.

### CODE REVIEW

**Comment by PrinceNebulon** [comment]
Aggregate query
* Group by options should only display valid options. I'm getting the error "Requested groupBy dimensions are not supported for aggregate queries in scope FLOW", but it doesn't say which ones are invalid. 
* Same thing for metrics. Getting the same type of error as above for some metrics.

Flow Observation query
* I can't figure out how to make a valid request and there's no documentation on the dev center about this resource. Can you shed some light?

**Comment by IdreesInc** [comment]
It seems that the values for metrics and groupBy in all analytics queries that require them are obtained using the "analyticsValueService". In each query the service provides all the possible values even when those values aren't all usable in the query, and the error provided gives the possible queries. Should I leave these as is then?

File: app/components/user-aggregate-query.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default Ember.Component.extend({
 	interval: null,
 	groupBy: ['userId'],
 	selectedMetrics: [],
-	userMetrics: ['tSystemPresence', 'tOrganizationPresence', 'tAgentRoutingStatus'],
+	// userMetrics: ['tSystemPresence', 'tOrganizationPresence', 'tAgentRoutingStatus'],
```
Comment (PrinceNebulon) [inline app/components/user-aggregate-query.js:10]: Is this being populated elsewhere now? If so, the commented code should be removed.

File: app/services/analytics-value-service.js [hunk 2]
```diff
@@ -180,20 +385,95 @@ export default Ember.Service.extend({
 				return;
 			}
 
-			var dimensions = swagger.definitions.AnalyticsQueryPredicate.properties.dimension.enum;
-			this.dimensions.clear();
-			this.dimensions.pushObjects(dimensions.sort());
+			// console.dir(swagger, {depth: null});
```
Comment (PrinceNebulon) [inline app/services/analytics-value-service.js:388]: Remove commented out code if it's no longer needed.

**Pull request merged**